setup.py

```diff
@@ -25,7 +25,7 @@
     install_requires=[
         "beautifulsoup4>=4.8",
         "html5lib>=1.0.1",
-        "pillow>=5.3",
+        "pillow>=7.1",
```

Comment:
Why are the version numbers different in setup.py and requirements.txt?

Reply:
requirements.txt specify the dev environment, which should preferably be close to the latest, whereas setup.py specifies the _minimum_ requirement for someone installing the WTL package. They don't neccessarily need the very latest.